pkgs/development/python-modules/confusable-homoglyphs/default.nix

```
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  setuptools,
+  unittestCheckHook,
+  wheel,
+  click,
+}:
+
+let
+  kebabToSnake = str: builtins.replaceStrings [ "-" ] [ "_" ] str;
+in
+buildPythonPackage rec {
+  pname = "confusable-homoglyphs";
+  version = "3.3.1";
```

```suggestion
  version = "3.3.1";
  pyproject = true;
```
Please add `pyproject = true;` to avoid the legacy setuptools build.

>  When unset, the legacy setuptools hooks are used for backwards compatibility.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-parameters-buildpythonpackage-parameters

---

AFAIK, it seems like this project uses a `setup.py`. There is no `pyproject.toml` to be found: https://git.sr.ht/~valhalla/confusable_homoglyphs/tree

---

thanks!